src/os/ui/featureedit.js

```diff
@@ -1674,6 +1676,63 @@ os.ui.FeatureEditCtrl.prototype.onAxisChange = function() {
 };
 
 
+/**
+ * Enables the modify geometry interaction.
+ *
+ * @export
+ */
+os.ui.FeatureEditCtrl.prototype.modifyGeometry = function() {
+  if (this.interaction) {
+    os.MapContainer.getInstance().getMap().removeInteraction(this.interaction);
+    this.interaction.setActive(false);
+    goog.dispose(this.interaction);
+  }
+
+  if (this.previewFeature) {
+    const mc = os.MapContainer.getInstance();
+    this.interaction = new os.interaction.Modify(this.previewFeature);
```

Comment:
We should define `this.interaction` in the constructor so the compiler is aware of the expected type.